app/hotels/src/navigation/allHotels/SearchResultsScreen.js

```diff
@@ -6,23 +6,18 @@ import {
   type NavigationType,
   useSetNavigationParams,
 } from '@kiwicom/mobile-navigation';
-import {
-  LayoutDoubleColumn,
-  StyleSheet,
-  Icon,
-  Translation,
-} from '@kiwicom/mobile-shared';
+import { LayoutDoubleColumn, StyleSheet } from '@kiwicom/mobile-shared';
 import { defaultTokens } from '@kiwicom/mobile-orbit';
 import { noop } from '@kiwicom/mobile-utils';
 
 import NewAllHotels from '../../allHotels/NewAllHotels';
-import HotelsNavigationOptions from '../HotelsNavigationOptions';
 import {
   SearchResultsContext,
   type SearchResultState,
 } from './SearchResultsContext';
 import SingleHotelContainer from '../../singleHotel/SingleHotelContainer';
 import { HotelsContext, type HotelsContextState } from '../../HotelsContext';
+import NavigationOptions from './navigtionOptions/NavigationOptions';
```

Comment:
Could you rename it to `navigationOptions/NavigationOptions`, just to fix the typo?